cmd/check.go

```diff
@@ -338,6 +341,13 @@ func check() {
 			log.Fatalf("Cannot marshal intermediate result: %v", err)
 		}
 
+		// build a report to build the updated context for the report index
+		report, err := reports.NewReport(manifest, rc)
```

Comment:
Possible optimization: We don't really need the whole report here. We could create here the `ReportSummary`.

But that's is not a reason to block this PR.

Reply:
Yeah... I did wonder about this... it's not ideal.